src/lib/Components/Countdown/DurationProvider.tsx

```diff
@@ -74,9 +73,8 @@ export class DurationProvider extends React.Component<Props, State> {
   render() {
     const { timeLeftInMilliseconds } = this.state
     const { timeOffsetInMilliseconds, children } = this.props
-
     return React.cloneElement(children, {
-      duration: moment.duration(timeLeftInMilliseconds + (timeOffsetInMilliseconds || 0)),
+      duration: moment.duration(timeLeftInMilliseconds + (timeOffsetInMilliseconds || 0) || 0),
```

Comment:
What are the `||` operators here for?

Reply:
The first is because `timeOffsetInMilliseconds` might not exist (calculates the difference between local time and gravity system time).

The second is because the timestamp passed into this (`startAt`) might be null, which would make the `timeLeftInMilliseconds` a `NaN`, and this entire `moment.duration` object invalid.

There are a bunch of ways to skin this cat, but this change basically ensures that _whatever happens_, we won't ever be showing `NaN`s in our countdown timer.

Currently, if the date was invalid or in the past, our timers would show `NaNm NaNd NaNh NaNm`. After this change, invalid/past dates will show `00m 00d 00h 00m`.